### ISSUE TITLE
Missing parameter type error is reported twice

### ISSUE THREAD

**Issue by not-my-profile** [microsoft/pyright#3078]
For the following code:

```python
# pyright: strict

def foo(x):
    pass
```

Pyright reports two errors:
```
  /tmp/foo.py:3:9 - error: Type of parameter "x" is unknown (reportUnknownParameterType)
  /tmp/foo.py:3:9 - error: Type annotation is missing for parameter "x" (reportMissingParameterType)
```

Unless I am missing something, I think a missing parameter type will also always be unknown?

If that is the case it might make sense to set `reportMissingParameterType` to `none` for `strict` mode, since it's already covered by `reportUnknownParameterType`?

The main reason I find this annoying is that when you want to leave one parameter type unknown in strict mode, you have to disable both diagnostic rules:

```python
# pyright: strict

def foo(x): # pyright: reportMissingParameterType=false, reportUnknownParameterType=false
    pass
```

(Yes I know I can disable it in `pyrightconfig.json` myself but I'd argue that the default behavior should be more ergonomic.)

**Comment by erictraut** [comment]
This is working as designed. The `reportUnknownParameterType` and `reportMissingParameterType` are different diagnostic checks, and they can be enabled or disabled separately. Or one could be set to 'error' and the other to 'warning', etc. 

If you are using strict mode, why would you possibly want to leave an input parameter unannotated???? That doesn't make sense. I would recommend at least adding an `Any` annotation if your intent is that it accepts any type.

**Comment by not-my-profile** [comment]
>If you are using strict mode, why would you possibly want to leave an input parameter unannotated?

I am defining a function to be passed to be passed to an API of another library. The parameter in question does not have a type annotation in the library, while there is a type annotation in typeshed stub, I am reluctant to use a type alias that's only defined in a stub in my code, since it's not actually part of the API. I could use `Any` but that would be wrong since it does not actually accept any type. What would you do recommend in that situation?
